docs/mkdocs.yml

```diff
@@ -58,6 +58,7 @@ nav:
       - dev/benchmark.md
 
 markdown_extensions:
+  - admonition
```

Comment:
Duplicated. It is already included below.

Reply:
Sorry,
I just noticed that `pymdownx.superfences` is also duplicated in the list. I will fix that 👍🏼